README.md

```diff
@@ -36,7 +36,7 @@ A starting point can be our [default dashboard](./dashboards/rustiq.json).  To d
 use a [Grafana pie chart plugin](https://grafana.com/grafana/plugins/grafana-piechart-panel/). It
 needs to be installed in order for the pie charts to be displayed.
 
-<img src="https://imgur.com/a/Bi5xurZ" alt="" />
+<img src="https://imgur.com/a/Bi5xurZ" width="600" alt="" />
```

Comment:
Hmm, I got a `Non-Image content-type returned` on this part.
![image](https://user-images.githubusercontent.com/12746683/120111150-67c92e80-c1a3-11eb-9710-b46ba070c83f.png)